pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdSources.java

```diff
@@ -119,7 +119,7 @@ void run() throws Exception {
                 System.err.println();
                 String chosenCommand = jcommander.getParsedCommand();
                 getUsageFormatter().usage(chosenCommand);
-                return;
+                throw e;
```

Comment:
Q: Have you tested whether the throw will print the exception stack trace? Since line 118 has already printed the error info.